### DISCUSSION TITLE
Creating a screen grab with a keypress

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#626] [Q&A]
Like the last question. No idea where to post this, move it as appropriate (Notice a trend here?).

I'm finding the inbuilt screenshot utility less than useful. Oftentimes I want to grab screen shots showing menu's and such, and that's all but impossible with the screenshot utility.

I've tried the PrtScr thing, and when I press PrtScr, the screen "blinks" as though something is happening, but when I try to paste into GIMP, it tells me there is nothing to paste.

And why doesn't the inbuilt screenshot utility grab the dock? I'm also finding that "Copy to Cliboard" is inop.

Any thoughts about this would be greatly appreciated.

chris

**Comment by teamcons** [comment]
Posting about the screenshot utility would be best in the screenshot repository
because i do believe it is about the screenshot

for menus i set the delay. 3 seconds, select area - i select approximatively the area, then open menus to show

the screen blink, works well when in a videogame because it does not disrupt it - outside of it it is pretty much "did it do something ?" - others first freeze then ask question. It may be nice to adapt behavior depending on whether fullscreen application running

for copying to clipboard you need to click on "copy to clipboard" - but that needs the utility interface. Im not sure how well it works under wayland though. It would indeed be nice to have it by default
